File: 012)CatchTheOnePiece/app/src/main/java/com/example/catchtheonepiece/OnePieceCharacterReply.java

package com.example.catchtheonepiece;

import java.util.List;
import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;


public class OnePieceCharacterReply {

    @SerializedName("characters")
    @Expose
    private List<Character> characters;
    @SerializedName("success")
    @Expose
    private Integer success;

    public List<Character> getCharacters() {
        return characters;
    }

    public void setCharacters(List<Character> characters) {
        this.characters = characters;
    }

    public Integer getSuccess() {
        return success;
    }

    public void setSuccess(Integer success) {
        this.success = success;
    }

}

File: 053)CatchTheOnePieceKotlin/app/src/main/java/com/elyesasimsek/catchtheonepiecekotlin/OnePieceCharacterReply.kt
package com.elyesasimsek.catchtheonepiecekotlin

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

data class OnePieceCharacterReply (
    @SerializedName("characters")
    @Expose
    val characters: List<Character>? = null,

    @SerializedName("success")
    @Expose
    val success: Int? = null
)